draft-reddy-tls-composite-mldsa.md

```diff
@@ -78,7 +78,9 @@ Unlike previous migrations between cryptographic algorithms, the decision of whe
 
 Cautious implementers may opt to combine cryptographic algorithms in such a way that an attacker would need to break all of them simultaneously to compromise the protected data. These mechanisms are referred to as Post-Quantum/Traditional (PQ/T) Hybrids {{I-D.ietf-pquip-pqt-hybrid-terminology}}. 
 
-Certain jurisdictions are already recommending or mandating that PQC lattice schemes be used exclusively within a PQ/T hybrid framework. The use of Composite scheme provides a straightforward implementation of hybrid solutions compatible with (and advocated by) some governments and cybersecurity agencies {{BSI2021}}.
+One practical way to implement a hybrid signature scheme is through a composite signature algorithm. In this approach, the composite signature consists of two signature components, each produced by a different signature algorithm. A composite key is treated as a single key that performs a single cryptographic operation such as key generation, signing, verification, encapsulation, or decapsulation by using its internal sequence of component keys as if they form a single key.
```

Comment:
I'd drop the mention "encapsulation or decapsulation"; we're not discussing hybrids in general, but specifically hybrid signature algorithms

Reply:
Good point, fixed.